src/_mixins.scss

```diff
@@ -142,6 +142,14 @@ $toplevel-border-color: rgba(black, 0.2);
     }
 }
 
+@function etched-shadow() {
```

Comment:
Kind of seems like premature optimization to put this here when it's only used in `_windows.scss`

Reply:
Yeah I went back and forth. Since I was using it in multiple places in that file (for text and icons), I pulled it into mixins, but I can just define this function in the windows file itself.